test/filesystem.test

```diff
@@ -36,6 +36,14 @@ test_expect_success mkfs_f2fs,sload_f2fs,fsck_f2fs "f2fs" "
 	fsck.f2fs images/test.f2fs
 "
 
+exec_test_set_prereq mkfs.btrfs
+exec_test_set_prereq sload.btrfs
+exec_test_set_prereq fsck.btrfs
+test_expect_success mkfs_btrfs,fsck_btrfs "btrfs" "
+	run_genimage_root btrfs.config test.btrfs &&
+	fsck.btrfs images/test.btrfs
```

Comment:
`fsck.btrfs` does not actually do anything, so just drop it here. I suppose we cannot actually validate the result...